### PR TITLE
Rebar3 as primary build tool

### PR DESCRIPTION
**Pull request by zmstone**
`erlang.mk` is still supported, but not used in travis-ci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,11 @@ sudo: required
 services:
   - docker
 
+# install latest rebar3 because the one travis provids doesn't support cover_export_name option
```
Comment (k32) [inline .travis.yml]: s/provids/provides/

File: .travis.yml [hunk 2]
```diff
@@ -5,8 +5,11 @@ sudo: required
 services:
   - docker
 
+# install latest rebar3 because the one travis provides doesn't support cover_export_name option
 before_install:
+  - git clone https://github.com/erlang/rebar3.git; cd rebar3; ./bootstrap; sudo mv rebar3 /usr/local/bin/; cd ..
```
Comment (k32) [inline .travis.yml:10]: Would be nice to checkout certain commit instead of master

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Brod - Apache Kafka Erlang client library
+[![Build Status](https://travis-ci.org/klarna/brod.svg)](https://travis-ci.org/klarna/brod) [![Coverage Status](https://coveralls.io/repos/github/klarna/brod/badge.svg?branch=master)](https://coveralls.io/github/klarna/brod?branch=master)
 
-Brod is an erlang implementation of the Apache Kafka protocol, providing support for both producers and consumers.
+# Brod - Apache Kafka Client for Erlang/Elixir
 
-[![Build Status](https://travis-ci.org/klarna/brod.svg)](https://travis-ci.org/klarna/brod)
+Brod is an erlang implementation of the Apache Kafka protocol, providing support for both producers and consumers.
```
Comment (k32) [inline README.md]: Nit: capitalize Erlang

**Review by k32 — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (zmstone) [reply .travis.yml:10]: this is only used in travis. no big deal if it breaks.

File: src/brod_utils.erl [hunk 4]
```diff
@@ -635,6 +659,10 @@ foldl_batch(Fun, Acc, [Msg | Rest]) ->
            end,
   foldl_batch(Fun, NewAcc, Rest).
 
+is_batch(X) ->
```
Comment (k32) [inline src/brod_utils.erl]: Nit: pattern matching would look nicer than chain of checks

**Review by k32 — APPROVED** [review]

**Review by k32 — APPROVED** [review]

**Pull request merged**